hvplot/converter.py

```diff
@@ -148,6 +148,12 @@ class HoloViewsConverter:
     hover_cols (default=[]): list or str
         Additional columns to add to the hover tool or 'all' which will
         includes all columns (including indexes if use_index is True).
+    hover_tooltips list[str] or list[tuple]:
+        A list of dimensions to be displayed in the hover tooltip.
+    hover_formatters:
+        A dict of formatting options for the hover tooltip.
+    hover_mode (default='mouse'):
+        The hover mode determines how the hover tool is activated.
```

Comment:
Maybe have a link to Bokeh and/or HoloViews documentation. 

For `hover_mode` maybe add all the options that are available.